### ISSUE TITLE
Restoring a backup after installing broken after 4.72.5

### ISSUE THREAD

**Issue by nalyDylan** [signalapp/Signal-Android#10128]
- [ x ] I have searched open and closed issues for duplicates
- [ x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x ] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
After installing Signal from android, the app crashes on Signal versions > 4.72.5 (4.72.5 works)
(Specifically I tried 4.72.5, 4.74.1, 4.74.3, and a beta APK for 4.75.0)

### Steps to reproduce
- install Signal from app store
- select "Restore Backup"

**Actual result:** App crashes
**Expected result:** The option to restore a backup

### Device info

**Device:** Moto Edge 5G
**Android version:** 10
**Signal version:** >4.72.5

### Link to debug log
N/A because I'm not far enough in the app, but here is the logcat output:
https://pastebin.com/1P458148

I put it in pastebin because GitHub mangles the output in the preview

**Comment by nalyDylan** [comment]
Expanding a thought since it's the OPEN_DOCUMENT intent, I tried installing another file manager app to make sure it wasn't failing from a file manager not being available? I have Simple File Manager, Amaze File Manager, and Astro File Manager all installed right now. I also tried some other apps that open files to make sure it was just Signal having this issue. Then I tried some other apps on my phone that probably use the same intent, like opening a keepass database in KeePassDX, and SyncThing seems to also work fine
